docs/IAssetToken.md

```diff
@@ -236,9 +242,17 @@ Will trigger `tokensToSend` and `tokensReceived` hooks as per ERC777.
 
 Will revert if `amount` doesn't respect the granularity.
 
-Will revert if the caller is not an operator of the token holder Will revert if the operator or the token holder are not allowed (blacklisted or not yet whitelisted) Will revert if the contract is in frozen state after an `emergencyStop`.
+Will revert if the caller is not an operator of the token holder.
+
+Will revert if the operator or the token holder are not allowed (blacklisted or not yet whitelisted).

```

Comment:
>Will revert if the operator or the token holder are not allowed to use the token contract (i.e. blacklisted or not  whitelisted if the contract is initialised in a list mode of operation).

What do you think? Is that a useful clarification or unnecessary?

Reply:
Useful :)